_2020/data-wrangling.md

```diff
@@ -308,7 +308,7 @@ leave that as an exercise to the reader.
 
 ## Analyzing data
 
-You can do math! For example, add the numbers on each line together:
+You can do the math! For example, add the numbers on each line together:
```

Comment:
This one is actually correct the way it was! It was intended as a "you can do math (in bash)" :sweat_smile: